src/composer.ts

```diff
@@ -520,6 +521,35 @@ export class Composer<C extends Context> implements MiddlewareObj<C> {
         return this.filter(Context.has.inlineQuery(trigger), ...middleware);
     }
 
+    /**
+     * Registers middleware for InlineQueryResults by the given id or ids.
+     * Telegram sends an inline query to your bot whenever a user types
```

Comment:
I'm not entirely happy with this description yet. First it tells you something about InlineQueryResult, then it changes the topic explaining what Inline queries are and then about the response. I think it should be in a different order. The introducing sentence, then it does something for the chosen results, then a brief "chosen results of what, chosen results of inline queries".

Not sure if my "this currently has a strange order" feeling is well enough explained with this comment?

Reply:
How about something like this?
`* Registers middleware for the ChosenInlineResult by the given id or ids.
     * ChosenInlineResult represents a result of an inline query that was
     * chosen by the user and sent to their chat partner. Check out
     * https://core.telegram.org/bots/api#choseninlineresult to read more
     * about chosen inline results.`